x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/thread_intelligence_analyst.ts

```diff
@@ -17,7 +17,7 @@ export const getThreadIntelligenceAnalyst: () => Omit<Role, 'name'> = () => {
         ...noResponseActionsRole.kibana[0],
```

Comment:
nit: Can you please also rename this to `...threatIntelligence...`, everywhere. Also the file name to `threat_intelligence_analyst`

Reply:
of course! 9ccea2d and f3558c0